app/javascript/utils/hooks/useFeatureFlag.js

```diff
@@ -2,7 +2,7 @@ import { useFlag as useFlagUnleash, useFlagsStatus } from '@unleash/proxy-client
 
 // If you want to prevent a user from changing this feature flag via the URL, you can add it to the urlBlockList set.
 // When set, the feature flag will only be determined by the Unleash API.
-const urlBlockList = new Set([''])
+const urlWhiteList = new Set(['test-partners', 'testFlag'])
```

Comment:
question: probably not in scope, but have we discussed why it would be preferred to put the white list in source code, as opposed to env vars for example?

Reply:
We haven't - I think that's a good topic for whatever discussion we have about managing feature flags